src/components/date_picker/super_date_picker/quick_select_popover/quick_select_popover.tsx

```diff
@@ -101,10 +101,16 @@ export class EuiQuickSelectPopover extends Component<
           end={end}
           prevQuickSelect={prevQuickSelect}
         />
+        {commonlyUsedRanges.length ? (
+          <EuiHorizontalRule margin="s" />
+        ) : undefined}
         <EuiCommonlyUsedTimeRanges
           applyTime={this.applyTime}
           commonlyUsedRanges={commonlyUsedRanges}
         />
+        {recentlyUsedRanges.length ? (
+          <EuiHorizontalRule margin="s" />
+        ) : undefined}
```

Comment:
Super optional comment, if you'd prefer a `&&` over a ternary I believe you can do

```suggestion
        {recentlyUsedRanges.length > 0 && (
          <EuiHorizontalRule margin="s" />
        )}
```

Feel free to collapse if you're fine with ternaries though!

Reply:
Yeah I guess that was my way of ensuring it didn't spit out the value of `recentlyUsedRanges.length`. I can change to `&&`